test/sol5/kyberMatchingEngine.js

```diff
@@ -2172,110 +2172,22 @@ contract('KyberMatchingEngine', function(accounts) {
                 });
             });
 
-            describe("test fees = BPS", async() => {
```

Comment:
why removing this?

Reply:
change to update the expected output is revert

Reply:
great thnaks